docs/en_US/SupportedFramework_Library.md

```diff
@@ -0,0 +1,44 @@
+# Framework and Library Supported 
+NNI provides Python API, supporting all framework models and libraries written in python (`version >= 3.5`), and we have implemented many related examples and detailed their principles and running steps.
+
+
+
+## Support Framework:
```

Comment:
For examples?

Reply:
You mean by merging the framework with the library, and then writing a general title:For Example?